Dockerfile

```diff
@@ -1,17 +1,33 @@
-FROM sameersbn/ubuntu:14.04.20170110
+FROM ubuntu:xenial
 MAINTAINER sameer@damagehead.com
 
-ENV GITLAB_CI_MULTI_RUNNER_VERSION=1.1.4 \
+
+RUN echo 'APT::Install-Recommends 0;' >> /etc/apt/apt.conf.d/01norecommends \
+ && echo 'APT::Install-Suggests 0;' >> /etc/apt/apt.conf.d/01norecommends \
+ && apt-get update \
+ && DEBIAN_FRONTEND=noninteractive apt-get install -y \
+      vim.tiny wget sudo net-tools ca-certificates unzip git openssh-client curl libapparmor1
+
```

Comment:
by reverting to `sameersbn/ubuntu:14.04.20170110` these lines will not be required